Forms/StartForm/LoadFileButton.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Windows.Forms;
+using CraphModel;
+using Serializing;
+using System.Collections.Generic;
+using GraphModelDraw;
+using System.IO;
+
+
+
+namespace Forms
+{
+    class LoadFileButton : Button
+    {
+
+        private DeserializeGraph deserializeGraph;
+        public Graph LoadGraph { get; set; }
+
+        private StartForm.StartForm startForm;
+
+        private List<VertexDraw> vertexDraws;
+
+        private List<EdgeDraw> edgeDraws;
+
+        private StartForm.DrawForm drawForm;
+
+        private FileInfo fI = new FileInfo("Graphs.json");
```

Comment:
It is better to make the filename permanent, as it is used in several places in the project.